### PR TITLE
Ioceans

### PR DESCRIPTION
**Pull request by Florie25**

### CODE REVIEW

File: _posts/2019-10-29-Subnero-at-iOcean-Sanya.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+date:  2019-10-29
+categories: news
+thumbnail: images/pulse-thumbnail-smartsea-team.jpg
+---
+
+Subnero was delighted to be with [Wuxi Smartsea Technology](http://www.smartseatech.com/en/index.aspx){:target="_blank"} at iOceans 19.
+
+The event took place from the 25th to the 27th of October, in Sanya, China.
+
+Details of the exhibition can be found [here](http://www.cso.org.cn/ggxx/2019/0926/2816.html){:target="_blank"}.
```
Comment (notthetup) [inline _posts/2019-10-29-Subnero-at-iOcean-Sanya.md]: Can we remove this sentence and move this hyperlink onto "The event" in the previous line. Or some other part of the post which is relevant to the link?

File: _posts/2019-10-29-Subnero-at-iOcean-Sanya.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+This is Subnero's first exhibition in China, with their new business partner.
```
Comment (notthetup) [inline _posts/2019-10-29-Subnero-at-iOcean-Sanya.md:14]: "with their new business partner Wuxi Smartsea Technology" ?

File: _posts/2019-10-29-Subnero-at-iOcean-Sanya.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+Subnero was delighted to be with [Wuxi Smartsea Technology](http://www.smartseatech.com/en/index.aspx){:target="_blank"} at iOceans 19.
+
+The event took place from the 25th to the 27th of October, in Sanya, China.
+
+Details of the exhibition can be found [here](http://www.cso.org.cn/ggxx/2019/0926/2816.html){:target="_blank"}.
+
+This is Subnero's first exhibition in China, with their new business partner. 
+
+"We are confident that this is the beginning of a long and successful partnership in China."
```
Comment (notthetup) [inline _posts/2019-10-29-Subnero-at-iOcean-Sanya.md:16]: We can quote in markdown using the "greater-than sign" `>`. Also may be nice to denote who the quote is from?

File: _posts/2019-10-29-Subnero-at-iOcean-Sanya.md (hunk 1 above)
Comment (Florie25) [reply _posts/2019-10-29-Subnero-at-iOcean-Sanya.md]: Done.

**Review by notthetup — APPROVED** [review]

**Comment by AnneThKu** [comment]
image

**Review by AnneThKu — APPROVED** [review]
Images have been re-arranged.

**Pull request merged**